kedro-telemetry/README.md

```diff
@@ -20,42 +20,39 @@ for further information on the intent of the data collection and what data is co
 For technical information on how the telemetry collection works, you can browse
 [the source code of `kedro-telemetry`](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-telemetry).
 
-## How do I consent to the use of Kedro-Telemetry?
+## How do I withdraw consent?
 
-Kedro-Telemetry is a Python plugin. To install it:
+To withdraw consent, you have a few options:
 
-```console
-pip install kedro-telemetry
-```
+1. **Set Environment Variables**:
```

Comment:
Shall we also add some explanation on what happens if several options are applied at the same time, how they redefine each other?

Relates to: https://github.com/kedro-org/kedro-plugins/issues/785

Reply:
Good catch with that bug. I added a description about the priority of the environment variables and overriding the `.telemetry` file. I believe we will fix the bug.

Reply:
Yep, let's wait until the fix and then revisit this paragraph to make sure they align.